### PR TITLE
solved the redirection issue

### PR DESCRIPTION
**Pull request by creativeadi**
Before:

https://github.com/Its-Aman-Yadav/Community-Site/assets/153895524/bd311419-677e-467a-8ca6-0d01a781caf1

After:

https://github.com/Its-Aman-Yadav/Community-Site/assets/153895524/5727dd95-df61-4dd4-8682-c174c2930bca

### CODE REVIEW

**Review by Its-Aman-Yadav — CHANGES_REQUESTED** [review]
Please redirect them to our social handles

**Comment by creativeadi** [comment]
@Its-Aman-Yadav  kindly provide with the respective social media handles

**Comment by Its-Aman-Yadav** [comment]
here - https://www.linkedin.com/company/open-source-village/?viewAsMember=true
https://twitter.com/OpenSourceVill

File: index.html [hunk 1]
```diff
@@ -214,8 +214,8 @@ <h2>Open Source Village</h2>
 
 
         <ul class="social-icon">
-          <li><a href="#" class="fa fa-twitter wow bounceIn" data-wow-delay="0.9s"></a></li>
-          <li><a href="#" class="fa fa-instagram wow bounceIn" data-wow-delay="1.2s"></a></li>
+          <li><a href="https://www.twitter.com" target="_blank" class="fa fa-twitter wow bounceIn" data-wow-delay="0.9s"></a></li>
```
Comment (Its-Aman-Yadav) [inline index.html:217]: Change to our social media handles

**Comment by creativeadi** [comment]
Uploading Screen Recording 2024-05-12 at 2.14.46 PM.mov…
@Its-Aman-Yadav 

fixed the redirection issue. the click on the social media handles directs the user to the official twitter and linkedin handles of this community. 

under GSSOC24

File: index.html (hunk 1 above)
Comment (creativeadi) [reply index.html]: fixed the changes as asked!

**Pull request closed without merging**